pandas/tests/extension/base/casting.py

```diff
@@ -43,8 +45,19 @@ def test_astype_str(self, data):
         expected = pd.Series([str(x) for x in data[:5]], dtype=str)
         self.assert_series_equal(result, expected)
 
+    @pytest.mark.parametrize(
+        "nullable_string_dtype",
+        [
+            "string",
+            pytest.param(
+                "arrow_string", marks=td.skip_if_no("pyarrow", min_version="1.0.0")
+            ),
+        ],
+    )
     def test_astype_string(self, data, nullable_string_dtype):
         # GH-33465
+        from pandas.core.arrays.string_arrow import ArrowStringDtype  # noqa: F401
```

Comment:
why is this needed?

Reply:
To force register the dtype, I think (the same reason that @simonjayhawkins also has been adding it in several places). AFAIU, once the ArrowStringDtype is merged with StringDtype, that will be done automatically and all those imports can be removed.

Reply:
yep. these are removed in #39908

Reply:
why can't you do the import IN the fixture itself? having in the test is very odd (when you *also* have the fixture)

Reply:
There is no fixture to import it inside, only a `pytest.mark.parametrize` decorator

Reply:
the `nullable_string_dtype` fixtures, sure there is

Reply:
we do this type of thing in the parquet tests, need to do the same here

Reply:
Can you be more specific as "this type of thing in the parquet tests" ?

> the nullable_string_dtype fixtures, sure there is

The whole point of this PR is to not use that, please look at the diff. I am removing the `nullable_string_dtype` fixture with a local parametrization

Reply:
why, this is *exactly* the reason this fixture should exist. to avoid having to do a local import like this. which will surely be done incorrectly the next time. a fixture exists for exactly this reason.

Reply:
The local imports are only temporary, they will be removed in https://github.com/pandas-dev/pandas/pull/39908

Reply:
ahh ok great